src/lib/tasks.ts

```diff
@@ -136,3 +137,27 @@ export async function installDeps(projectPath: string) {
     await execaCommand(installCommand, { cwd: projectPath });
   }
 }
+
+export function checkCargoVersion(minimalVersion: string, invalidVersionsList: string[]) {
+  const regex = /cargo-contract-contract (.*)-unknown-x86_64-unknown-linux-gnu/;
```

Comment:
This regex do not work on Mac where `cargo-contract --version` output looks like this:

`cargo-contract 4.0.0-rc.1-unknown-aarch64-apple-darwin`

Make it compatible for all architecture